superbench/benchmarks/micro_benchmarks/cuda_memory_bw_performance.py

```diff
@@ -58,38 +43,16 @@ def _preprocess(self):
         if not super()._preprocess():
             return False
 
-        # Format the arguments
-        if not isinstance(self._args.mem_type, list):
-            self._args.mem_type = [self._args.mem_type]
-        self._args.mem_type = [p.lower() for p in self._args.mem_type]
-
         # Check the arguments and generate the commands
         for mem_type in self._args.mem_type:
```

Comment:
you didn't declare `mem_type ` in `add_parser_arguments()`, but used it in `_preprocess()`, it's weird.